### PR TITLE
QUIZ-55 Create github actions workflow

### PR DESCRIPTION
**Pull request by BenLanglois**
Runs build and lint scripts when a PR is created or updated.

### CODE REVIEW

**Review by alexgirard — APPROVED** [review]
👍

File: src/components/TopBar.tsx [hunk 1]
```diff
@@ -14,9 +14,8 @@ const TopBarContainer = styled.div`
 
 const TopBarContent = styled(LeftAlignRow)`
   width: 100%;
-  height: 5em;
   background-image: url(assets/blobs/headerBlob.png);
-  background-size: cover;
+  background-size: 100% 180%;
```
Comment (heathermusson) [inline src/components/TopBar.tsx:18]: Why are we changing css? Can we move this to a separate PR with an explanation please

Comment (alexgirard) [reply src/components/TopBar.tsx:18]: I think this was changed in #17 which was merged so we can remove this line?

Comment (BenLanglois) [reply src/components/TopBar.tsx:18]: #17 got merged into this branch, which is why this change is showing here.
